src/test/perf_counters.cc

```diff
@@ -16,12 +16,15 @@
                            // now, this include has to come before the others.
 
 
+#include "common/Clock.h"
```

Comment:
Changes to this file seems unrelated to this PR

Reply:
You're right, sorry about that. Updated.